api/data-source/data-source.ts

```diff
@@ -82,5 +84,10 @@ const addOrder = (order: any) => {
   return order;
 };
 
-export const ordersMock = { orders, addOrder };
-export const eventsMock = { events, addEvent, updateEvent };
+const updateOrder = (order: any, id: string, index: string | number) => {
+  (<any>orders)[index] = { id, ...order };
+  return (<any>orders)[index];
+};
+
+export const ordersDataSource = { orders, addOrder, updateOrder };
```

Comment:
separate `ordersDataSource` and `eventsDataSource`

```js
// data-sources/events-data-source.js
export const eventsDataSource = {
  // ...
}

// data-sources/orders-data-source.js
export const eventsDataSource = {
  // ...
}
```

add an `index.js` file to export both event and orders data sources.

```js
// data-sources/index.js
export { eventsDataSource } from './events-data-source';
export { ordersDataSource } from './orders-data-source';
```